### ISSUE TITLE
wsdl:part messages should use XML name from the referenced element

### ISSUE THREAD

**Issue by silverspace** [fiorix/wsdl2go#111]
I am interfacing with a WSDL-based SOAP service that worked with an older version of your wsdl2go library. When I upgraded from 8f16131 to c0bfd34, our client started generating an XML request that the server now rejects. The reason for this is due to <wsdl:part> messages using the part name attribute instead of the referenced element's name.

Here is the relevant <wsdl:part> message snippet from our WSDL file:

<wsdl:message name="IELDSubmissionService_Submit_InputMessage">
  <wsdl:part name="parameters" element="tns:Submit"/>
</wsdl:message>
The current wsdl2go master XML output (rejected by server): <parameters><data><ELDIdentifier>...</parameters>.

The older wsdl2go output (accepted by server): <Submit xmlns="http://www.fmcsa.dot.gov/schemas/FMCSA.ELD.Infrastructure"><data><ELDIdentifier>...</Submit>

In other words, the XML tag is now parameters instead of Submit for the above WSDL snippet:

type OperationIELDSubmissionService_Submit_InputMessage struct {
	Parameters *Submit `xml:"parameters,omitempty" json:"parameters,omitempty" yaml:"parameters,omitempty"`
}
I took a shot at a PR that fixes my particular use case, but I don't know enough about SOAP to know what the proper behavior for all <wsdl:part> tags should be. Proposed fix for this issue is at:

https://github.com/fiorix/wsdl2go/pull/110

**Comment by fiorix** [comment]
Merged.
